04-Multi-Band-Raster.Rmd

```diff
@@ -294,30 +307,74 @@ plotRGB(RGB_stack,
 plotRGB(RGB_stack,
         r = 1, g = 2, b = 3, 
         scale=800,
-        stretch = "Lin")
+        stretch = "lin")
+```
 
+In the first image the forest was a nice bright green, however, seeing individual
+features was a bit of the challenge.  In the second plot with `stretch="lin"`, 
+the forest isn't as pretty green but individual features are easier to see. In 
+the upper left corner an area of uniform green can now be differentiated into 
+seperate trees or shrubs. 
 
-```
+>Bonus: Here we did a linear ("lin") stretch of the values. You can also stretch 
```

Comment:
Removed in accurate info.  Issue https://github.com/data-lessons/NEON-R-Spatial-Raster/issues/29